co-operative.md

```diff
@@ -30,48 +30,59 @@ You can contact us the following ways:
 - 🔗 [hypha.coop](https://hypha.coop)
 - 📧 hello@hypha.coop
 
-## Member-owners and Employees
+## Members and Employees
 
-Every **member-owner** is both an employee and co-owner of Hypha. There are 
-seven member-owners, all of which are founding employees, with four Directors.
+Every **member-worker**, or **member** in short, is both a member and an employee of Hypha.
+There are seven members, all of which are founding members, with six also serving as directors.
 
-A **member-owner**:
+A **member**:
   - Has completed a membership application (and/or been officially welcomed into 
     the membership)
-  - Is a *permanent* or *founding* employee
+  - Is a *founding*, or *permanent*, member if they are one of the seven members who founded Hypha
   - Is considered:
-    - *active* if they are in good standing with their membership fees and have 
-      not requested to change their status
+    - *active* if they have not requested to change their status
     - *inactive* if they have requested to change their status
       - Inactive members are unable to vote, serve as a director, participate in 
         projects, or work as an employee for so long as they remain so
       - Inactive members can request to become active at any time
 
 An **employee**: 
+  - Works under a *contract of service* with Hypha
```

Comment:
>It is clear that each organization calling itself a worker co-op must be assessed independently to determine whether there is a contract of service, i.e., employer/ employee relationship, or a contract for services, i.e., an independent contractor situation.

https://canadianworker.coop/guidelines-for-determining-whether-or-not-there-is-an-employer-employee-relationship-in-a-worker-co-op/